grpc/oob/bmc/bmc.go

```diff
@@ -207,27 +234,32 @@ func (m Action) UpdateUser(ctx context.Context) error {
 
 // DeleteUser functionality for machines.
 func (m Action) DeleteUser(ctx context.Context) error {
-	timer := prometheus.NewTimer(metrics.ActionDuration.With(prometheus.Labels{"service": "bmc", "action": "Delete_user"}))
+	timer := prometheus.NewTimer(metrics.ActionDuration.With(prometheus.Labels{"service": "bmc", "action": "delete_user"}))
 	defer timer.ObserveDuration()
 
+	tracer := otel.Tracer("pbnj")
+	ctx, span := tracer.Start(ctx, "client.DeleteUser")
+	defer span.End()
+
 	host, user, password, err := m.ParseAuth(m.DeleteUserRequest.Authn)
 	if err != nil {
 		return err
 	}
+	span.SetAttributes(attribute.String("bmc.host", host), attribute.String("bmc.username", user))
 
 	creds := &v1.UserCreds{Username: m.DeleteUserRequest.Username}
 	status := fmt.Sprintf("deleting user %q", creds.GetUsername())
 	m.SendStatusMessage(status)
 
 	actions, err := m.setupConnection(ctx, user, password, host, creds)
 	if err != nil {
-		m.SendStatusMessage("connectiion setup failed")
+		// setupConnection is responsible for sending a status message and updating the span
 		return err
 	}
 
-	if err = oob.DeleteUser(ctx, actions); err != nil {
-		m.SendStatusMessage(fmt.Sprintf("error %s: %v", status, err))
-		m.Log.Info(fmt.Sprintf("error %s: %v", status, err))
+	err = oob.DeleteUser(ctx, actions)
+	if err != nil {
+		m.noteError(fmt.Sprintf("error %s: %v", status, err), span)
```

Comment:
curious, what's the driving force to move away from the one line if statement?

Reply:
I think other lines had it split. I can move all three to one line.

Reply:
Cleaned up in all three locations.